src/main/java/seedu/address/logic/commands/AbstractEditCommand.java

```diff
@@ -85,15 +85,16 @@ private Person getPersonToEdit(Model model) throws CommandException {
     private CommandResult saveEditedPerson(
             Model model,
             Person personToEdit,
-            Person editedPerson
+            Person editedPerson,
+            String successMessage
```

Comment:
nit: Might not be a need for making successMessage a param

Reply:
This is one of the limitations when the derived class relies heavily on the parent. Will review in upcoming iterations.